src/tokenizer.rs

```diff
@@ -666,10 +673,26 @@ fn advance_number(code: &str) -> Advancement {
     })
 }
 
+#[inline]
+fn parse_identifier(code: &str) -> IResult<&str, &str> {
+    recognize(pair(
+        // Identifiers must start with at least 1 alphabetic character
+        alpha1,
+        // And then they must be followed by 0 or more alphanumeric (or '_') characters
+        take_while(|x| is_alphanumeric(x as u8) || x == '_'),
```

Comment:
This could be replaced with `x.is_ascii_alphanumeric()` -- https://doc.rust-lang.org/std/primitive.char.html#method.is_ascii_alphanumeric